tests/mr/device/mr_test.hpp

```diff
@@ -233,7 +232,7 @@ struct mr_factory {
 struct mr_test : public ::testing::TestWithParam<mr_factory> {
   void SetUp() override
   {
-    if (GetParam().name == "CUDA_Async" && should_skip_async()) {
+    if (GetParam().name == "CUDA_Async" && !rmm::mr::cuda_async_memory_resource::is_supported()) {
```

Comment:
Wouldn't it be more robust to put it in the `make_async` factory?

Reply:
What would it return?

Reply:
Not sure if this is better, but please take a look.